connectivity/check/context.go

```diff
@@ -293,6 +293,16 @@ func (ct *ConnectivityTest) GetTest(name string) (*Test, error) {
 	panic("missing test descriptor for a registered name")
 }
 
+// GetTestOrDie returns the test scope for test named "name" if found,
+// or panics otherwise.
+func (ct *ConnectivityTest) GetTestOrDie(name string) *Test {
```

Comment:
nit: the upstream convention for functions that panic on failure is usually `MustFoo` (see e.g. [`netip.MustParseAddr`](https://pkg.go.dev/net/netip#MustParseAddr)), so this could be named `MustGetTest` instead.

Reply:
✨ https://github.com/cilium/cilium-cli/pull/2098 ✨